Templates/Files/MS SQL/MS SQL database SELECT to CSV file/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "MS SQL database SELECT to CSV file",
+  "version": "0.1.1",
```

Comment:
Release version should be 1.0. In the process in Frends you can use 0.x until we get all the things correct.